packages/core-mobile/app/services/network/NetworkService.ts

```diff
@@ -187,6 +194,81 @@ class NetworkService {
       return acc
     }, {} as Networks)
   }
+
+  private async fetchDeBankNetworks(): Promise<Networks> {
```

Comment:
Most of the logic in this function is because Glacier doesn't provide info about these chains, so here i'm translating deBank's chain info to our Network format.